eng/code-quality-reports/src/main/resources/index.html

```diff
@@ -517,6 +517,39 @@ <h2>Azure Java SDK</h2>
                     </li>
                 </div>
             </ul>
+
+            <ul class="list-group components">
+                <li class="list-group-item group-name">
+                    <strong>Test report</strong>
+                </li>
+                <div class="group-items">
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-tux"></ion-icon>
+                        <a href="../test-coverage-Linux-1.11/test-coverage/index.html" target="_blank" class="links">Linux - Java LTS</a>
+                    </li>
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-tux"></ion-icon>
+                        <a href="../test-coverage-Linux-1.8/test-coverage/index.html" target="_blank" class="links">Linux - Java 8</a>
+                    </li>
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-windows"></ion-icon>
+                        <a href="../test-coverage-Windows-1.11/test-coverage/index.html" target="_blank" class="links">Windows - Java LTS</a>
+                    </li>
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-windows"></ion-icon>
+                        <a href="../test-coverage-Windows-1.8/test-coverage/index.html" target="_blank" class="links">Windows - Java 8</a>
+                    </li>
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-apple"></ion-icon>
+                        <a href="../test-coverage-macOS-1.11/test-coverage/index.html" target="_blank" class="links">MacOS - Java LTS</a>
+                    </li>
+                    <li class="list-group-item sub-component">
+                        <ion-icon name="logo-apple"></ion-icon>
+                        <a href="../test-coverage-macOS-1.8/test-coverage/index.html" target="_blank" class="links">MacOS - Java 8</a>
+                    </li>
```

Comment:
The test coverage should basically be the same for all platforms. I would rather we just pick one and only link to that.

Reply:
Updated with a single report.

Reply:
I'm also not sure just yet what the actual path will be to the test reports, maybe we should make this change in another PR after we have merged this one, and after we have the reports being pushed to Blob storage